recipes/magics/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "2.34.1" %}
+
+package:
+  name: magics
+  version: {{ version }}
+
+source:
+  fn: Magics-{{ version }}-Source.tar.gz
+  url: https://software.ecmwf.int/wiki/download/attachments/3473464/Magics-{{ version }}-Source.tar.gz
+  sha256: 8df27f8f262ebc32a61f8696df15a7b4a6e4203b2a8e53fe7aa13caa1c4e3fa4
+
+build:
+  number: 0
+  skip: True  # [win or py3k]
+  detect_binary_files_with_prefix: true
+
+requirements:
+  build:
+    - cmake
+    - swig
+    - boost 1.63.*
+    - proj.4
```

Comment:
Apologies. I assumed you had experience with conda-forge pinnings :flushed: 

Take a look at https://github.com/conda-forge/conda-forge.github.io/blob/master/scripts/pin_the_slow_way.py#L39-L82

(I am working on better docs for this one.)